src/main/java/org/folio/rest/validator/PackagePutBodyValidator.java

```diff
@@ -34,11 +34,7 @@ public void validate(PackagePutRequest request) {
     String value = attributes.getPackageToken() != null ? attributes.getPackageToken().getValue() : null;
 
     if (isSelected == null || !isSelected) {
-      ValidatorUtil.checkFalseOrNull("allowKbToAddTitles", allowKbToAddTitles);
```

Comment:
Deleted all this checks (and tests for it), because we can never reach this case. 
We were throwing an error when isCustom and isSelected both are false. 
So I remove this checks and transferred throwing here

Reply:
I will open another task for this